recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
         - def_snprintf.patch  # [win]
 
 build:
-    number: 3
+    number: 1
```

Comment:
I am confused about this. Shouldn't `3` be the chosen one instead of `1`? Anyways, you mentioned that fixed the issue for you in another package so let's try this then...